docs/turn-howto.md

```diff
@@ -145,6 +149,12 @@ This will install and start a systemd service called `coturn`.
     We recommend that you only try to set up TLS/DTLS once you have set up a
     basic installation and got it working.
 
+    NB: If your TLS certificate was provided by Let's Encrypt, TLS/DTLS will
+    not work with any Matrix client that uses Chromium's WebRTC library. This
```

Comment:
Could we have a link to the underlying webrtc issue here? From the EA and EiOS issues you linked, this seems to be https://bugs.chromium.org/p/webrtc/issues/detail?id=11710

Reply:
Done.